saleor/static/dashboard-next/categories/queries.ts

```diff
@@ -100,16 +100,16 @@ export const categoryDetails = gql`
           node {
             id
             name
-            availability {
+            basePrice {
+              amount
+              currency
+            }
+            pricing {
```

Comment:
`pricing.available` is there for compatibility but it's also deprecated. We should use `product.isAvailable` for that purpose.